src/library/scala/collection/BitSetLike.scala

```diff
@@ -204,6 +204,27 @@ trait BitSetLike[+This <: BitSetLike[This] with SortedSet[Int]] extends SortedSe
   def subsetOf(other: BitSet): Boolean =
     (0 until nwords) forall (idx => (this.word(idx) & ~ other.word(idx)) == 0L)
 
+  override def head: Int = {
```

Comment:
Generally, scanning bit by bit is slower than using `numberOfTrailingZeros` unless the bits are pretty dense.  It might be worth investigating whether `foreach` could be sped up, and if so, to make a separate commit for it.